common/crypto/platform/openssl/crypto.cpp

```diff
@@ -93,6 +93,45 @@ string GetOpenSSLErrorMessage() {
 	return errorDescription;
 }
 
+// We need our own custom callback routine, as the default one will prompt for a
+// passphrase.
+int PrivateKey::CapturePrivateKeyPassphrase(char *buf, int size, int rwflag, void *u) {
+	// We'll only use this callback for reading passphrases, not for writing
+	// them.
+	assert(rwflag == 0); 
```

Comment:
Observation:

This is fine. It would also be possible to wrap this function also, so that it's not possible to pass in anything other than `0`

Reply:
Ref. our F2F discussion, this is resolved